src/Breadcrumb/index.js

```diff
@@ -6,9 +6,17 @@ import { LinkContainer } from 'react-router-bootstrap';
 const CustomBreadcrumb = (props) => {
 
   const breadcrumbs = props.items.map((item) => {
+
+    const handleClick = () => {
+      if (item.name.includes('Project:')) {
+        props.handleClick();
+      }
+    };
+
+    console.log(item);
```

Comment:
Can probs remove the console.log

```suggestion
    console.log(item);
```